Classes/common/CDTIncrementalStore.m

```diff
@@ -2208,6 +2221,75 @@ - (id)executeSaveRequest:(NSSaveChangesRequest *)saveRequest
     return @[];
 }
 
+- (id)executeBatchUpdateRequest:(NSBatchUpdateRequest *)updateRequest
```

Comment:
HMMMM.  So what I really want to return here is an NSBatchUpdateRequest ... but I can't, because that class doesn't provide setters for fields I need to set.  So CDTISBatchUpdateResult is my hack.  It looks just like NSBatchUpdateRequest, but I can set the fields I need.  So I'd rather return this as an id.  I suppose I should capture this in a comment in the code.